scripts/generator/graphql_generator/csharp/SDK/ResponseMergeUtil.cs.erb

```diff
@@ -0,0 +1,69 @@
+namespace <%= namespace %>.SDK {
+    using System;
+    using System.Collections;
+    using System.Collections.Generic;
+
+    public class ResponseMergeUtil {
+        public delegate void MergeFieldDelegate(string field, IDictionary into, IDictionary responseA, IDictionary responseB);
+
+        public static void DoNotMergeIfExists(string field, IDictionary into, IDictionary responseA, IDictionary responseB) {
+            if (!responseA.Contains(field)) {
+                into[field] = responseB[field];
+            }
+        }
+
+        public static void MergeOverwrite(string field, IDictionary into, IDictionary responseA, IDictionary responseB) {
+            into[field] = responseB[field];
+        }
+
+        public static void MergeListsConcat(string field, IDictionary into, IDictionary responseA, IDictionary responseB) {
+            List<object> newList = new List<object>();
+            IList listA = (IList) responseA[field];
+            IList listB = (IList) responseB[field];
+
+            foreach(object item in listA) {
```

Comment:
Good catch :+1: previously I had `newList` be and `IList` so this is a left over from this.